intera_interface/src/intera_motion_interface/motion_trajectory.py

```diff
@@ -73,19 +73,19 @@ def __init__(self, label = None,
     def stop_trajectory(self):
         self._client.stop_trajectory()
 
-    def send_trajectory(self, wait_for_result=True):
+    def send_trajectory(self, wait_for_result=True, timeout=None):
```

Comment:
Add documentation for the new parameter
`@param timeout: maximum time to wait for trajectory result. If timeout is reached, return None.`